src/tablecloth/time/utils/datatypes.clj

```diff
@@ -0,0 +1,21 @@
+(ns tablecloth.time.utils.datatypes
+  (:require [tech.v3.datatype :refer [elemwise-datatype]]
+            [tech.v3.datatype.datetime.packing :as datetime-packing]
+            [tech.v3.datatype.datetime.base :as datetime-base]
+            [clojure.set :refer [union]]))
+
+(defn get-datatype
+  "Returns the datatype keyword of `data` as identified by dtype-next's
+  type system. If it is not a known type, it will return `:object`. `data`
+  can be a column (or some type of 'reader' as defined by dtype-next) or
+  an individual element of data."
+  [data]
+  (elemwise-datatype data))
```

Comment:
Let's wrap the `elemwise-datatype` fn to make things a bit more readable? Or maybe this is confusing?